### ISSUE TITLE
Any plans for Linux support? 

### ISSUE THREAD

**Issue by m-rey** [mrmekon/connectr#40]
I was wondering if there are plans of supporting Linux in the future

**Comment by mrmekon** [comment]
Yes, sort of.  I recently switched back to linux, and definitely want Connectr ported.  No clue when I'll get around to actually doing it, but it will surely happen sooner or later.
